com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -227,7 +227,7 @@ private void BuildDestinationToTransitionInfoTable()
         }
 
         /// <summary>
-        /// Creates the
```

Comment:
ugh? That was missing like this??

Reply:
Yes, that week was a bit rough for me (family issue) and I made some really bad mistakes. 
😞